client/app/index.tsx

```diff
@@ -1,8 +1,11 @@
 
 import React from 'react'
 import { Redirect } from 'expo-router'
+import { LogBox } from 'react-native';
```

Comment:
Should we remove this now?

Reply:
I think this could be good to keep as it stops the annoying pop ups if someone is testing the app?